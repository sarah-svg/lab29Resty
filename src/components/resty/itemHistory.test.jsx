import React from 'react';
import { render, cleanup } from '@testing-library/react';
import HistoryItem from './ItemHistory';

describe('Iten History', () => {
  afterEach(() => cleanup());
  it('renders HistoryItem', () => {
    const { asFragment } = render(<HistoryItem
      method="GET"
      url="www.google.com"
      id= "1"
    />);    
        
    expect(asFragment()).toMatchSnapshot();
  });
});
  

