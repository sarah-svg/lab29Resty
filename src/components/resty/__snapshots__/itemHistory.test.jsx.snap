// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Item History renders HistoryItem 1`] = `
<DocumentFragment>
  <article
    id="1"
  >
    <p
      id="1"
    >
      GET
    </p>
    <span
      id="1"
    >
      www.google.com
    </span>
  </article>
</DocumentFragment>
`;
